Steepfile

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+D = Steep::Diagnostic
+
+target :lib do
+  signature "sig"
+
+  check "lib"
+
+  ignore "lib/prism/serialize.rb"
+  ignore "lib/prism/ffi.rb"
```

Comment:
Does this file need to be ignored? If it's just for making progress and we want to come back to it later that's fine, let's just notate it with a TODO.